### PR TITLE
Fix for the infinite drag bug

### PR DESCRIPTION
**Pull request by raelbr**
Fix for bug related at: https://github.com/jgwhite/ember-sortable/issues/86

### CODE REVIEW

**Comment by jgwhite** [comment]
The tests are failing right now due to a breaking changes in Ember 2.5 beta. I need to fix ember-sortable’s acceptance tests. Thanks for the PR and thanks for your patience!

**Comment by jgwhite** [comment]
Hey sorry for the delay on this — still looking for a solution for the failures on beta/canary.

**Comment by raelbr** [comment]
It's ok! For my project i reopened the addon with the modifications to solve the bug untill the official update. So, no rush! I just hope this change help other people too and have no unexpected side-effects. And thanks again for your help!

**Comment by jgwhite** [comment]
I’ve fixed the issue with our drag helper. Could you rebase on top of master?

**Comment by jgwhite** [comment]
This looks good to me. Could you squash into one commit with a descriptive commit message containing the phrase “closes #86”? Thanks!

**Comment by jgwhite** [comment]
@raelbr ping :)

**Comment by raelbr** [comment]
I'm sorry, as it's my first contribuition i'm a little confused on how to squash it into one single commit... tried some tutorial but i think something went wrong =S

**Comment by jgwhite** [comment]
@raelbr no problem! Here’s the process:

```sh
git remote add upstream git@github.com:jgwhite/ember-sortable
git fetch upstream
git rebase -i upstream/master
```

This will give you a UI that looks like this:

```
pick 92d152d closes #86
pick 9087cff closes #86
pick 48906e8 fix for the infinite drag bug
```

Edit it to look like this:

```
reword 92d152d closes #86
fixup 9087cff closes #86
fixup 48906e8 fix for the infinite drag bug
```

Then save and the rebase will run and pause to allow you to reword the first commit. It'd suggest rewording the first commit to match the name of this PR: “Fix for the infinite drag bug”

Then finally:


```
git push origin master -f
```

**Comment by raelbr** [comment]
Hello @jgwhite. Is everything fine now? =)

**Comment by jgwhite** [comment]
Almost :) Looks like you squashed onto the last commit on master and you’re not getting your proper credit. I think the best bet is to run:

```
git commit --amend --author "Your name goes here <your email goes here>"
```

**Comment by raelbr** [comment]
@jgwhite, Done! Is it ok now? =)

**Comment by jgwhite** [comment]
👍

**Comment by raelbr** [comment]
Nice! Thank you for you patience, @jgwhite !
